sdk/identity/Azure.Identity/src/AuthorizationCodeCredential.cs

```diff
@@ -85,8 +85,8 @@ internal AuthorizationCodeCredential(string tenantId, string clientId, string cl
             Argument.AssertNotNull(authorizationCode, nameof(authorizationCode));
             _clientId = clientId;
             _authCode = authorizationCode ;
-            _options = options ??= new TokenCredentialOptions();
-            _pipeline = CredentialPipeline.GetInstance(options);
+            _allowMultiTenantAuthentication = options?.AllowMultiTenantAuthentication ?? true;
+            _pipeline = CredentialPipeline.GetInstance(options ?? new TokenCredentialOptions());
```

Comment:
passing null as opposed to a `TokenCredentialOptions` uses the singleton instance of `CredentialPipeline` this would change that behavior.

Reply:
I may be missing something, but I believe this should have the same behavior as what is there currently:

https://github.com/Azure/azure-sdk-for-net/blob/658dbd4976d088a21f0fc368b6ed42e5688200cd/sdk/identity/Azure.Identity/src/AuthorizationCodeCredential.cs#L88-L89